src/repository/models.py

```diff
@@ -401,6 +402,14 @@ class Preprint(models.Model):
     date_updated = models.DateTimeField(blank=True, null=True)
     current_step = models.IntegerField(default=1)
 
+    article = models.OneToOneField(
```

Comment:
This means I can only submit the preprint to a single journal, which presumably it is okay?

Reply:
Yep.